frontend/src/streamlist/StreamsTable.jsx

```diff
@@ -83,6 +89,10 @@ export default class StreamsTable extends React.Component {
   }
 
   getColumnInfo() {
+    const { isSimple } = this.props;
+    if (isSimple) {
+      return this.simpleColumnInfo;
+    }
     return this.columnInfo;
```

Comment:
I feel like this is kind of an antipattern. Like passing a boolean to a function which makes the function do two wildly different things. But I can't think of a way to do it differently without making a whole other component with a lot of code copied from here.

Anyways, you can simplify this to `return isSimple ? this.simpleColumnInfo : this.columnInfo;`